packages/ag-charts-community/src/util/async.ts

```diff
@@ -8,7 +8,7 @@ export class AsyncAwaitQueue {
                 resolve(true);
             };
             const timeoutFn = () => {
-                const queueIndex = this.queue.findIndex(successFn);
+                const queueIndex = this.queue.indexOf(successFn);
```

Comment:
?

Reply:
successFn is the item we're trying to find, so `Array.findIndex()` is the wrong thing to be doing here.